filtered-resource-service/src/main/java/uk/gov/gchq/palisade/service/filteredresource/web/config/EndpointConfiguration.java

```diff
@@ -48,22 +61,49 @@ AkkaHttpServer akkaHttpServer(final ServerProperties properties, final Collectio
         return new AkkaHttpServer(hostname, properties.getPort(), routeSuppliers);
     }
 
+    /**
+     * Route for "/resource/{token}" to the {@link WebsocketEventService}
+     *
+     * @param websocketEventService the websocket event service to connect to
+     * @param objectMapper          an object mapper for object serialisation
+     * @return the websocket router
+     */
     @Bean
     WebsocketRouter websocketRouter(final WebsocketEventService websocketEventService, final ObjectMapper objectMapper) {
         return new WebsocketRouter(websocketEventService, objectMapper);
     }
 
+    /**
+     * Route for "/api/{topic}" to the {@link KafkaProducerService}
+     *
+     * @param kafkaProducerService the kafka producer service to connect to
+     * @return the kafka rest writer router
```

Comment:
Might need a bit of clarification here, as fun as "the kafka rest writer router" is to say, its not super descriptive